sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -0,0 +1,119 @@
+﻿# Level-Zero backend specification
+
+## 1. Introduction
+
+This extension introduces Level-Zero backend for SYCL.
+It is built on top of Level-Zero runtime enabled with [Level-Zero API](https://spec.oneapi.com/level-zero/latest/index.html).
+The Level-Zero backend is aimed to provide the best possible performance of SYCL application on a variety of targets supported.
+The currently suppported targets are all Intel GPUs starting with Gen9.

```

Comment:
```suggestion
The currently supported targets are all Intel GPUs starting with Gen9.
```

Suggest you run spell check on this document.  :-)

Reply:
Fixed. Surprisingly, the spell-checker did not highlight anything else.